frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -53,6 +53,7 @@ export const OperatorHubSubscribeForm = withFormState((props: OperatorHubSubscri
 
   const submit = () => {
     const catalogSourceConfig = props.catalogSourceConfig.data.find(csc => csc.metadata.name === OPERATOR_HUB_CSC_NAME);
+    const hasBeenEnabled = !_.isEmpty(catalogSourceConfig) && _.includes(catalogSourceConfig.spec.packages.split(','), packageName);
```

Comment:
Really should look at having the CRD use an actual array of packages instead of a comma separated list.